commands/deploy.go

```diff
@@ -60,6 +60,16 @@ func (c *DeployCmd) Run() (err error) {
 	}
 	fmt.Printf("Packaging app in: %s\n", dir)
 
+	errs := IsValidNodeApp(dir)
+	if len(errs) > 0 {
+		var se []string
+		for _, err := range errs {
+			se = append(se, fmt.Sprintf("- %s", err))
+		}
+		errstr := strings.Join(se, "\n")
+		return fmt.Errorf("not a valid Node.js app: \n\n%s", errstr)
+	}
```

Comment:
I think that we should print only one error at a time. The user will fix only one error at a time anyway.

Reply:
I originally thought this too. 

On reflection, it can be frustrating being drip-fed errors. 

I want to give this a try, and see what users have to say about it.